clothing/dashboard_views.py

```diff
@@ -47,3 +51,34 @@ def get_context_data(self, **kwargs):
             ))
         )
         return context
+
+
+class OrderAggregatedView(ClothingAppMixin, TemplateView):
+    template_name = "clothing/dashboard/orders_aggregated.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data()
+
+        types = Type.objects.all()
+        colors = Color.objects.all()
+        sizes = Size.objects.all()
+
+        context['orders'] = OrderedDict()
+        for additional_type, additional_name in [(False, _('Kostenlos')), (True, _('Zusätzlich'))]:
+            # Create bins
+            context['orders'][additional_name] = OrderedDict()
```

Comment:
Better sort types, colors, sizes once you fetch them from the db and use normal dicts here.